cmd/ddev/cmd/config.go

```diff
@@ -215,8 +215,10 @@ func init() {
 	ConfigCommand.Flags().StringVar(&projectTypeArg, "project-type", "", projectTypeUsage)
 	ConfigCommand.Flags().StringVar(&phpVersionArg, "php-version", "", "The version of PHP that will be enabled in the web container")
 	ConfigCommand.Flags().StringVar(&httpPortArg, "http-port", "", "The router HTTP port for this project")
-	ConfigCommand.Flags().StringVar(&httpPortArg, "router-http-port", "", "The router HTTP port for this project")
-	ConfigCommand.Flags().StringVar(&httpsPortArg, "https-port", "", "The router HTTPS port for this project")
+	ConfigCommand.Flags().StringVar(&httpPortArg, "router-http-port", "", "The router HTTP port for this project; alias for `--router-http-port`")
```

Comment:
Am I reading this wrong? It's an alias for itself?

Reply:
Thanks, I put the text on the wrong one. http-port is alias for router-http-port, https-port is alias for router-https-port. You're awesome.

Reply:
Ahh that makes more sense 😅